### PR TITLE
implementation added

### PR DESCRIPTION
**Pull request by PawelYak**

### CODE REVIEW

**Comment by ziobron** [comment]
✅ CI Passed - Task: fibonacci
🏅 Granted 6 XP for @PawelYak

**Comment by ziobron** [comment]
⏱ 3 XP extra for @PawelYak for on-time delivery

File: homework/fibonacci/fibonacci.hpp [hunk 1]
```diff
@@ -1,11 +1,24 @@
 #pragma once
 
 int fibonacci_iterative(int sequence) {
-    // TODO: Your implementation goes here
-    return 0;
+  if (sequence != 0) {
+    auto b{1};
+    for (auto i = 1, a = 0; i < sequence; i++) {
```
Comment (nauka-programowania-MA) [inline homework/fibonacci/fibonacci.hpp:6]: please don't declare `a` inside for.

File: homework/fibonacci/fibonacci.hpp [hunk 2]
```diff
@@ -1,11 +1,24 @@
...
+  return 0;
 }
 
 int fibonacci_recursive(int sequence) {
-    // TODO: Your implementation goes here
+
+  if (sequence == 0) {
     return 0;
+  } else if (sequence == 1) {
+    return 1;
```
Comment (nauka-programowania-MA) [inline homework/fibonacci/fibonacci.hpp:20]: Sequence hold exactly the same value as you want to return for 0 and 1 :)

Better write:

`if (sequence < 2) {
    return sequence;
}`

File: homework/fibonacci/fibonacci.hpp [hunk 3]
```diff
@@ -1,11 +1,24 @@
...
+  }
```
Comment (nauka-programowania-MA) [inline homework/fibonacci/fibonacci.hpp:21]: If you return variable here, you don't need to use `else if` or `else`

**Review by nauka-programowania-MA — APPROVED** [review]

**Pull request closed without merging**